src/commands/postDeploy/checkLinuxWebAppDownDetector.ts

```diff
@@ -9,11 +9,12 @@ import { KuduClient } from "vscode-azurekudu";
 import { DeployResult } from "vscode-azurekudu/lib/models";
```

Comment:
This isn't directly related to this PR, but in your demo the other day I saw this message and it bothered me:
> "Diagnosing web app "${node.root.client.siteName}" for critical errors..."

It makes it seem like I already have critical errors even on a successful deploy. It also makes it feel like I have to wait for the diagnostics to finish before I can truly know if my site deployed, but we never "finish" diagnostics, we just time out. Can we just get rid of the message?

Another reason is because you might not actually be diagnosing if FileLogging isn't enabled.

Reply:
Yeah, I totally get that.  And the message saying that it "timed out" also makes it sound like that there are critical errors, but we just couldn't find them.  I wanted some way to alert the user that more information may be incoming to keep their attention, but I think I agree with you.